assignment.js

```diff
@@ -65,6 +77,22 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+const values = [3, 48, 29, 12, 18]
+let currentLargestNumber = 0
+
+for (let i = 0; i < values.length; i++) {
```

Comment:
@nataliekk nice approach and great job getting this working. 

An alternative approach would have been to only use the `values[i] > currentLargestNumber` test to determine the largest number.  Take a look at the [approach used here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L98) for more details.